cmd/iconbridge/chain/bsc/verifier.go

```diff
@@ -166,7 +174,9 @@ func (vr *Verifier) Update(header *types.Header) (err error) {
 			return errors.Wrapf(err, "getValidatorMapFromHex %v", err)
 		}
 		// update validators only if epoch block and no error encountered
+		vr.prevValidators = vr.validators
 		vr.validators = newValidators
+		vr.useNewValidatorsFromHeight = (&big.Int{}).Add(header.Number, big.NewInt(1+int64(len(vr.prevValidators)/2)))
```

Comment:
Does this height need to be explicitly computed, or is the height part of some block data in the block where a validators change happens? If it's part of block data, then using the height from it instead of computing it manually is better. Otherwise, it's okay.